Makefile

```diff
@@ -11,6 +11,7 @@ GO=GOOS=linux GO111MODULE=on GOFLAGS=-mod=vendor go
 GO_BUILD_RECIPE=$(GO) build -o $(OUT_DIR)/$(PACKAGE_BIN) -ldflags '-X $(PACKAGE)/version.Version=$(REV)' $(PACKAGE_MAIN)
 GOFMT_CHECK=$(shell find . -not \( \( -wholename './.*' -o -wholename '*/vendor/*' \) -prune \) -name '*.go' | sort -u | xargs gofmt -s -l)
 REV=$(shell git describe --long --tags --match='v*' --always --dirty)
+GOARCH=amd64
```

Comment:
I'm not sure we need this change.
If we need to limit PAO builds to amd64 only, we will most likely need to fix this at prow level

Reply:
Agreed, this won't work for NTO.  I believe we had this in the past for NTO, but it got removed by the architecture team.

Reply:
OK. Lets remove this from there and check later if we should limit PAO builds at prow level.